File: Codeforces/Python/hulk.py
n = int(input(''))
it = 'it'
that = 'that'

if (n == 1):
    print("I hate it")

if (n > 1):
    for i in range(n):
        if (n % 2 == 0):
            print(f'I hate {that} I love {it}')
            break

        elif (n % 2 != 0):
            print(f'I hate {that} I love {that} I hate {it}')
            break

File: Codeforces/C++/hulk.cpp
#include<bits/stdc++.h>
using namespace std;

int main() {
    cin.tie(0);
    ios_base::sync_with_stdio(false);


    int n;
    while (cin>>n)
    {
        for (int i=1;i<=n;i++)
        {
            if (i%2)
               cout<<"I hate ";
            else
               cout<<"I love ";
            if (i==n)
               cout<<"it"<<endl;
            else
               cout<<"that ";
        }
    }
    return 0;

}